hooks/pre-command

```diff
@@ -10,14 +10,24 @@ main() {
 
   if [[ -n $role && -n $build ]]; then
     echo "~~~ Assuming IAM role $role ..."
-    eval "$(assume_role_credentials "$role" "$build" | credentials_json_to_shell_exports)"
+    output="$(assume_role_credentials "$role" "$build" | credentials_json_to_shell_exports)"
+    rc="$?"
```

Comment:
hrm, I don't think this rc check is necessary with `set -e` + `output=`, it's just `eval` that swallows any errors